.gitignore

```diff
@@ -25,7 +25,7 @@ codecov-report.json
 
 
 package-lock.json
-
+amplify-backup/
```

Comment:
Does this get created by the Amplify CLI?

Reply:
Yes, I gitignored it since our build was complaining about it's existence. Digging into the why now. I think ultimately it can stay in and I'll need to see about modifying the `amplify.yml`. :crossed_fingers: